src/AssetTransferApi.ts

```diff
@@ -364,7 +380,7 @@ export class AssetTransferApi {
 				txMethod = 'limitedReserveTransferAssets';
 				transaction = await limitedReserveTransferAssets(
 					_api,
-					xcmDirection,
+					xcmDirection as XcmDirection,
```

Comment:
I think we should avoid having to type cast here. I would assume its the `Direction.local` which is causing this issue, we might want to find a way to change that. More comments below.